src/qt/pivx/receivewidget.cpp

```diff
@@ -112,7 +112,7 @@ void ReceiveWidget::loadWalletModel(){
         refreshView();
 
         // data change
-        connect(this->addressTableModel, SIGNAL(dataChanged(QModelIndex,QModelIndex)), this, SLOT(refreshView()));
+        connect(this->addressTableModel, &AddressTableModel::dataChanged, [this]{ReceiveWidget::refreshView();});
```

Comment:
The `ReceiveWidget::refreshView()` is redundant, can just be `refreshView()`.

Reply:
This one had to ultimately be converted into a lambda due to #1385.